### DISCUSSION TITLE
Rename `FeatureTable` into `Layer`

### DISCUSSION THREAD

**Discussion by nyurik** [maplibre/maplibre-tile-spec#117] [General]
It seems the `FeatureTable` concept described in the spec is semantically identical to the MVT's `Layer`. If so, I think it would be better to use the same terminology to avoid confusion and to simplify adaption.  Or does the featuretable has any additional semantic meaning?

**Comment by 1ec5** [comment]
“Layer” is a familiar term, but it’s also a very overloaded term. I’m constantly struggling to distinguish between tile layers and style layers in conversation. (Look, they even rhyme when fully qualified!)

**Reply by andrewharvey** [reply]
+1 I'll usually refer to them explicitly as "tile layer"/"source layer" and "style layer" but I think within the context of the MLT spec here simply "layer" is fine since it'll only refer to a tile/source layer.

**Comment by kkaefer** [comment]
I think calling this "Layer" was a mistake in the original spec and it stems from the (invalid) assumption that each layer would only be rendered once. In the code, we have to use names like "SourceLayer" to distinguish it from the style layers.

**Reply by kkaefer** [reply]
Plus, "layer" doesn't actually describe the concept well. I think `FeatureTable`, `FeatureCollection` (like in GeoJSON)` or `Feautreset` would be better terms.

**Comment by nyurik** [comment]
Within the MLT context, we only deal with one type of layers. Feature table is a good term for a specific type of a layer, but I see no reason to restrict MLT to only support features - it could be routing tables, it could be raster images, etc.  A better lingo IMO could be `layer tag` (some enum value), and some of these values could be a "feature table"
